versions.sh

```diff
@@ -86,12 +86,39 @@ for version in "${versions[@]}"; do
 		jq <<<"$json" -c --argjson doc "$doc" '
 			.[env.version] = {
 				version: env.fullVersion,
-				variants: [ "apache-buster", "fpm-buster", "fpm-alpine3.12" ],
+				variants: [
+					# arrays of arrays to control ordering
+					if [ "7", "8.9", "9.1" ] | index(env.version) then
+						[
+							[ "buster" ],
+							[ "alpine3.13" ]
+						]
+					elif env.version | startswith("9.") then
+						[
+							[ "buster" ],
+							[ "alpine3.14", "alpine3.13" ]
+						]
+					else
+						[
+							[ "bullseye", "buster" ],
```

Comment:
Was this just for testing?  (I don't see any `bullseye` in the `versions.json` and we'll probably want to add it to all once `php` has it, right?)

Reply:
Yeah, was future proofing for Drupal 10 or something.